packages/legend-art/style/base/_panel.scss

```diff
@@ -77,6 +77,13 @@
     @include ellipsisTextOverflow;
   }
 
+  &__header__title__content__with__margin {
```

Comment:
This change to core style is not justifiable to me. Please create other classes and add to local stylesheet instead.

Reply:
Fixed! Moved additional styling to _file-generation-editor.

Reply:
Added configure for Morphir plugin which sets url fields.